pkg/es/config/config.go

```diff
@@ -188,10 +202,65 @@ func (c *Configuration) GetTagDotReplacement() string {
 }
 
 // GetConfigs wraps the configs to feed to the ElasticSearch client init
-func (c *Configuration) GetConfigs() []elastic.ClientOptionFunc {
+func (c *Configuration) GetConfigs(logger *zap.Logger) []elastic.ClientOptionFunc {
 	options := make([]elastic.ClientOptionFunc, 3)
+	if c.TLS.Enabled {
+		tlsConfig, err := c.CreateTLSConfig(logger)
+		if err != nil {
+			return nil
+		}
+		httpClient := &http.Client{
+			Timeout: c.Timeout,
```

Comment:
Seems like this is something extra, but I think it's ok to have it here.

Reply:
This is only being applied when TLS is enabled. Can you make it available for every case?

Reply:
@pavolloffay Yes. This is only being applied for httpclient which is being used by TLS case. I cannot find where to add for every case. we use `BulkProcessorService` which allows to easily process bulk requests. There is no interface to set timeout. Could you point me out more specifically? Thanks.

Reply:
So the driver does not use the client when tls is disabled?

Something like this would not work? 
```
// GetConfigs wraps the configs to feed to the ElasticSearch client init
func (c *Configuration) GetConfigs(logger *zap.Logger) []elastic.ClientOptionFunc {
	options := []elastic.ClientOptionFunc{elastic.SetURL(c.Servers...), elastic.SetSniff(c.Sniffer)}
	httpClient := &http.Client{
		Timeout: c.Timeout,
	}
	if c.TLS.Enabled {
		ctlsConfig, err := c.TLS.createTLSConfig(logger)
		if err != nil {
			return nil
		}
		httpClient.Transport = &http.Transport{
			TLSClientConfig:ctlsConfig,
		}
	} else {
		options = append(options, elastic.SetBasicAuth(c.Username, c.Password))
	}
	options = append(options, elastic.SetHttpClient(httpClient))
	return options
}
```